server/routes.ts

```diff
@@ -66,6 +66,15 @@ export const addRoutes = (
   app.get('/auth', function (req: Request, res: Response) {
     // Authenticate the request is from a valid user here
     const src = req.query.src as string
+    if (!config.host || !config.secret) {
```

Comment:
I suggest writing a validateConfig function that logs the error. It can accept `res` as an argument and return false if config is invalid. It should exit if the result is false. The validateConfig call be added to acquire embed session and generation token end points